package.json

```diff
@@ -8,7 +8,8 @@
     "build": "node scripts/build.js",
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
-    "fmt": "prettier --write ."
+    "fmt": "prettier --write .",
```

Comment:
this should rather be `fmt:fix` and the check should be `fmt`?

Reply:
Formatting is currently not covered by our [package-json-conventions](https://eslint.org/docs/latest/contribute/package-json-conventions).

@nzakas what do you think about script names?

Reply:
I'd suggest lets keep the script name to be `format` for analysing formatting problems. and `format:fix` for the write formatted changes. I can send a PR to add it to package conventions as well.

Reply:
`format` & `format:fix` would be aligned with `lint` & `lint:fix`, which looks good for consistency, but in the context of the word "format" it might be surprising that `format` only checks the format, so maybe `format:check` & `format:fix`?

Reply:
Since this PR has a high potential of running into merge conflicts with any future PRs, what do you think about merging this now as-is, and continuing this discussion on a PR that updates package conventions? Then, we could get back to this repo and update the script names if needed.

Reply:
Yea lets merge this PR and then pick this as a separate issue.